doc/stdenv/meta.chapter.md

```diff
@@ -125,17 +125,38 @@ $ cd path/to/nixpkgs
 $ nix-build -A your-package.tests
 ```
 
+Note that Hydra and [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review) don't build these derivations by default, but they are useful pointers for Nixpkgs' CI ([`@ofborg`](https://github.com/NixOS/ofborg)) that may help identify issues when changes such as version bumps are directly made to the package.
+
 #### Package tests {#var-meta-tests-packages}
 
-Tests that are part of the source package are often executed in the `installCheckPhase`.
+Tests that are part of the source package are often executed in the `installCheckPhase`. This phase is also suitable for performing a `--version` test for packages that support such flag. Here's an example:
+
+```nix
+# Say the package is git
+stdenv.mkDerivation(finalAttrs: {
+  pname = "git";
+  version = "...";
+  # ...
+
+  installCheckPhase = ''
```

Comment:
also add `doInstallCheck = true;` here as it is disabled by default

Reply:
> also add `doInstallCheck = true;` here as it is disabled by default

Thanks!